internal/commands/apply.go

```diff
@@ -95,7 +95,7 @@ var applyWaitFn = rollout.WaitUntilComplete // allow override in tests
 
 func doApply(ctx context.Context, args []string, config applyCommandConfig) error {
 	if len(args) != 1 {
-		return cmd.NewUsageError("exactly one environment required")
+		return cmd.NewUsageError(fmt.Sprintf("exactly one environment required, but provided: %v", args))
```

Comment:
use `%q` instead of `%v` everywhere so that output has quotes:

https://go.dev/play/p/CKLC6iCbczd

Reply:
Will do.